src/objects.rs

```diff
@@ -103,6 +103,38 @@ impl OsmId {
             _ => None,
         }
     }
+    /// Returns the inner id.
+    pub fn as_i64(&self) -> i64 {
+        match *self {
+            OsmId::Node(n) => n.0,
+            OsmId::Way(n) => n.0,
+            OsmId::Relation(n) => n.0,
+        }
+    }
+}
+
+/// Wrapper used for `StoreObjs` trait.
+pub enum OsmObjWrapper<'a> {
```

Comment:
Looks like reinventing `Cow`

Reply:
I don't need the copy part, that's why I wrote this enum.

Reply:
`OsmObj` is clone, so why adding code when standard tools exists?

Reply:
I don't want to clone anything either. The only difference between the two is that I recreate the object on one side and have a reference returned by the map on the other. If you still think that it's better to use `Cow` then I'll update. :)

Reply:
As `StoreObjs::get` is not needed in the trait, I think all of that can just be removed. Then each implementation can do whatever they want.

Reply:
Removed.